Dockerfile

```diff
@@ -1,37 +1,23 @@
-# SSL Certificate generation with let's encrypt
-
 # Before running this, your <domain> DNS should already point to your
-# AWS instance IP and your AWS security group should allow let's encrypt servers
+# AWS instance IP and you already generated a valid certificate for your DNS
```

Comment:
I think this kind of info should really appear in the README instead, I think most users won't even bother opening the Dockerfile and these steps are mandatory.

I'd remove all these comments and edit the README instead